### DISCUSSION TITLE
Courses

### DISCUSSION THREAD

**Discussion by annasuza** [swellstores/community#133] [Help] [answered]
Hi there, 

How would someone set up a course with Swell? 
Could this be done IN Swell or would it need to be done elsewhere with the help of Zapier?

Thanks so much. 

Susanna

**Comment by sags95** [comment]
Hi @annasuza,  the easiest way to get started would be by setting up digital products for your courses in Swell. This would cover everything regarding the ecommerce aspect for your courses (including checkout). Zapier would come in handy in order to automatically enrol your customers into certain courses within an e-learning platform. 

Here are a few useful Zaps that would be relevant:

- https://zapier.com/apps/cypher-learning/integrations
- https://zapier.com/apps/learningsuite/integrations

If you decide to use Zapier with Swell, just be sure to enable it through the Integrations page on the Swell dashboard.

**Reply by annasuza** [reply]
Are those the only course management softwares that you can be connected to?

**Reply by annasuza** [reply]
hi there, thank you'd

Is there way to actually create the course IN swell? Perhaps, maybe, with embedded videos? If there are worksheets included though, how would they be accessed?

I am just not sure how that would be available JUST to people who pay for it.

Sorry if this is all so obvious or if I'm overcomplicating it. Im used to WP and Woo, so it's hard to imagine simple.

**Reply by sags95** [reply, marked as answer]
When it comes to pre-built integrations, it would have to be ones that are connected through Zapier. Otherwise Swell can be integrated with any software as long as they provide a REST API, however you would have to build this integration.

As for creating the courses in Swell, I do see this as a possibility as well, however you would have to be familiar with web development.  One way would be to customize the product model to include a file attachment (or a collection of files), so you can attach your worksheets. Then when a customer purchases a course, maybe they would log into a customer portal and access this content.  You can find some useful info here regarding model customization: https://developers.swell.is/guides/data-model-customization

**Comment by annasuza** [comment]
Thank you. This is very helpful!!!
